fiftyone/utils/sam3.py

```
@@ -0,0 +1,550 @@
+            all_detections.extend(detections.detections)
+
+        return fol.Detections(detections=all_detections)
+
+    def _parse_output(self, output, width, height, label):
+        """Convert SAM3 output to FiftyOne Detections."""
+        masks = output.get("masks", [])
+        boxes = output.get("boxes", [])
+        scores = output.get("scores", [])
+
+        detections = []
+        for mask, box, score in zip(masks, boxes, scores):
```

Ruff (0.14.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)